include/zephyr/bluetooth/conn.h

```diff
@@ -335,14 +361,20 @@ struct bt_conn_info {
 	uint8_t role;
 	/** Which local identity the connection was created with */
 	uint8_t id;
+	/** Connection Flags. */
+	uint8_t flags;
```

Comment:
I guess these are the `BT_CONN_FLAG_` flags, right? 

Any reason why it isn't defined as an enum so that we can use the enum type here instead?